tests/strategies/test_abc.py

```diff
@@ -92,6 +93,15 @@ def test_get(
             return_json=testdata(strategy_name),
         )
 
+    if backend == "python" and strategy_name == "dataresource":
```

Comment:
This code block 'seems' like it is repeated a few times in other parts of the PR, any way of maybe making it more compact? [Not sure, sometimes you want to violate DRY anyway https://softwareengineering.stackexchange.com/questions/400183/what-should-i-consider-when-the-dry-and-kiss-principles-are-incompatible]

Reply:
It's a fast and easy fix, essentially, but in style with the rest of the test code base. I think over time we've come to actually test the same thing three times, unfortunately. But it might be in slightly different levels of minutae. In any case, I don't think anything other than a complete test code base rewrite will make this more compact :/